app/templates/brief.html

```diff
@@ -25,53 +25,41 @@
 {% if brief.status == 'closed' %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    {%
-      with
-      type = "temporary-message",
-      heading = "This opportunity is closed for applications",
-      message = "The deadline was {}".format(brief.applicationsClosedAt|dateformat)
-    %}
-      {% include "toolkit/notification-banner.html" %}
-    {% endwith %}
+    {{ dmBanner({
+      "title": "This opportunity is closed for applications",
+      "headingLevel": 2,
```

Comment:
The default heading level is 2, so we don't need to specify it.

Reply:
So it is! Will remove it